core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -187,7 +186,7 @@ class KafkaConfigTest {
     val advertisedHostName = "routable-host"
     val advertisedPort = 1234
 
-    val props = TestUtils.createBrokerConfig(0)
+    val props = createDefaultConfig()
```

Comment:
this `createDefaultConfig` does not return a valid config (https://github.com/apache/kafka/pull/18547/files#r1944140562)

@mingyen066 will fix it by a minor patch